packages/contentstack-launch/src/base-command.ts

```diff
@@ -105,7 +105,7 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
       this.flags['data-dir'] || this.flags.config
         ? this.flags.config || resolve(this.flags['data-dir'], config.configName)
         : resolve(process.cwd(), config.configName);
-    const baseUrl = config.launchBaseUrl || getLaunchHubUrl();
+    const baseUrl = config.launchBaseUrl || this.launchHubUrl;
```

Comment:
@harshithad0703 , what if baseUrl is undefined ? pls handle this condition. like below 
let baseUrl = config.launchBaseUrl || this.launchHubUrl;

if (!baseUrl) {
    baseUrl = getLaunchHubUrl();
}

Reply:
done